.github/actions/buildkite/action.yml

```diff
@@ -13,7 +13,7 @@ inputs:
     required: true
   secret:
     description: 'Vault secret with the token field.'
-    default: secret/observability-team/ci/buildkite-automation
+    default: secret/observability-team/ci/buildkite-automation-v2
```

Comment:
I create a new token with adjusted permissions.

The current token was created by Victor.

Reply:
Need to know what are the adjusted permissions 🙏